### DISCUSSION TITLE
START_PRINT macro doesn't except bed temperatures higher than 100°C

### DISCUSSION THREAD

**Discussion by rcmanslo** [Guilouz/Creality-Helper-Script-Wiki#507] [Issues]
Hi All.

I'm using latest Creality FW and helper scripts. And I can't set bed temperature higher than 100°C. I checked all installed scripts and I can't find where this max temperature is set. 
Printer.cfg has  following settings related to bed: 

[heater_bed]
heater_pin: PB10
sensor_type: EPCOS 100K B57560G104F
sensor_pin: PC4
#control: watermark
# control: pid
# pid_kp: 27
# pid_ki: 0.08
# pid_kd: 0
min_temp: 0
max_temp: 115
  
Can somebody point me to right direction ?

BR, Tadej

**Comment by Guilouz** [comment]
Max. heated bed temperature is 100 degres

**Reply by rcmanslo** [reply]
Hi.,

HW suports temperatures up to 115. And I have used 105 for asa printing without any problems. How can I change max temperature of bed.

Thank you for support.

Br Tadej

**Reply by Guilouz** [reply]
No HW support up to 100. It’s difficult to reach over.
115 is just a security setting, bed can’t reach this value.

**Reply by dingausmwald** [reply]
When setting in mainsail it raises to 110 easily. Soo...
its the same with ender printers. Official is 100, but 110 was reached easily and is recommended for ABS f. e. as mentioned across the internet. Even Orca slicers default bed temp for abs is 105.

Could you please make this editable or at least raise a bit? All this is out of warranty anyway.

**Comment by dingausmwald** [comment]
Found the code and hope i can post it here:
in /usr/share/klipper/klippy/extras you can mod the file heater_bed.py

this part sets the max temp set in printer.cfg (i guess) -15. So 115 -15 by default. Edit the max temp in printer.cfg or this file and restart.
    def cmd_M140(self, gcmd, wait=False):
        # Set Bed Temperature
        temp = gcmd.get_float('S', 0.)
        if temp > self.max_temp - 15.0:
            temp = self.max_temp - 15.0
        pheaters = self.printer.lookup_object('heaters')
        pheaters.set_temperature(self.heater, temp, wait)
